.github/workflows/validate-bicep.yaml

```diff
@@ -37,12 +37,13 @@ jobs:
         uses: actions/checkout@v4
       - name: Parse release version and set environment variables
         run: python ./.github/scripts/get_release_version.py
-      - uses: oras-project/setup-oras@main
-      - name: Download rad-bicep
+      - name: Setup Azure CLI
```

Comment:
why do we need azure CLI?

Reply:
I'm pretty sure the Bicep CLI relies on the az CLI for certain commands so both are needed